ffs/ffs_terms.md

```diff
@@ -0,0 +1,172 @@
+
+ANY SOFTWARE PRODUCED UNDER A FFS PROPOSAL IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#### (d) Dispute Resolution
+
+Any dispute arising out of, or in connection with, a FFS proposal, shall be referred to, and finally resolved by arbitration under the latest version of the FFS Arbitration Rules, which are deemed to be incorporated by reference. The number of arbitrators shall be three. The place of arbitration shall be Geneva, Switzerland. The award shall be deemed to have been made at the place of arbitration. The tribunal may conduct the arbitration at any location they consider appropriate, including, online through Mattermost, IRC, Slack, videoconferencing, or any other electronic communication mediums.
+
+#### (e) Governing Law
+
+Disputes shall be resolved by *ex aequo et bono*, in accordance with what is considered to be fair and equitable without regard to any applicable law.
+
+#### (f) Entire Agreement
+
+This document, and referenced sources, contain the entire agreement between the Donor and Contractor with respect to any matter related to a FFS proposal, and supersedes and replaces any prior agreements or understandings. These terms can only be modified in writing and digitally signed by the Contractor and all Donors that contributed to a FFS proposal.
+
+#### (g) Title to Contributions
+
+By donating XMR to a proposal, the Donor irrevocably gives, delivers, transfers and assigns to the Contractor all rights, title and interest in the property, commonly known as Monero (XMR), provided that the obligations under the FFS proposal are met. If a proposal fails to meet quorum under section 3(b), or the Contractor resigns or abandons a proposal, or if proposed changes to a proposal fails under section 4(c), and the proposal is relegated back to the Idea stage, title to said property shall pass to The Monero Project and the Core Team shall have the sole legal right to determine the property's disposition.
+
+#### (h) Severability
+
+If any part of these terms is declared unenforceable or invalid, the remainder will continue to be valid and enforceable.
```

Comment:
>enforceable

How and by whom?

Reply:
Either by the tribunal or court. Basically, this means that if one clause is considered invalid, this does not invalidate the whole contract.